recipe/meta.yaml

```
@@ -5,28 +5,32 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/x/xesmf/xesmf-{{ version }}.tar.gz
-  sha256: 66b6deb37f43749df06819fc2003bbd394207617cf7d8fd9dafe655d7af7b94c
+  url: https://github.com/JiaweiZhuang/xESMF/archive/v{{ version }}.tar.gz
```

```suggestion
  url: https://github.com/pangeo-data/xESMF/archive/v{{ version }}.tar.gz
```

Jiawei's repo has been abandoned for now.